.gitignore

```diff
@@ -2,3 +2,5 @@
 npm-debug.log
 node_modules/
 build/
+dist/
```

Comment:
`dist/` is not being created in any of the scripts(?)

Reply:
Oh, the dist/ was for electron-builder... not electron-packager... I will delete it...